core/src/wallet/common/AbstractWallet.cpp

```diff
@@ -241,15 +240,14 @@ namespace ledger {
         }
 
         FuturePtr<api::Account> AbstractWallet::getAccount(int32_t index) {
+            auto it = _accounts.find(index);
```

Comment:
Ah, this is a caching fix! Can you explain in the PR that you also fix that, pretty please? :)